fred/fred_commands/help.py

```diff
@@ -1,14 +1,16 @@
 from __future__ import annotations
 
-import logging
+import re
```

Comment:
why not regex here ?

Reply:
It wasn't performance critical, but we could use it for consistency

Reply:
Yeah I think we should